packages/eslint-config-ente/index.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+    extends: [
+        'next',
+        'turbo',
+        'prettier',
+        'plugin:@typescript-eslint/recommended',
+        'plugin:@typescript-eslint/recommended-requiring-type-checking',
```

Comment:
we had a different set of imports earlier. lets use that to stay consistent and not change the behaviour. 

also `prettier`  needs to be at the end for it to work properly.